spec/controllers/volunteers_controller_spec.rb

```diff
@@ -218,6 +218,22 @@
     it 'renders the index template' do
       expect(subject).to render_template :index
     end
+    it 'assigns header' do
+      subject
+      expect(assigns(:header)).to_not be_nil
+    end
+    it 'assigns volunteer, html' do
+      subject
+      expect(assigns(:volunteers)).to eq([volunteer])
+    end
+    it 'assigns volunteer, json' do
+      get :index, format: :json
+      expect(assigns(:volunteers)).to eq([volunteer])
+      expect(JSON.parse(response.body).length).to eq(1)
+      expect(JSON.parse(response.body)[0]['email']).to eq(volunteer.email)
+      expect(JSON.parse(response.body)[0]['name']).to eq(volunteer.name)
+      expect(JSON.parse(response.body)[0]['phone']).to eq(volunteer.phone)
```

Comment:
I think a cleaner way of doing this is `response_body = JSON.parse(response.body)` and then chaining off that

Reply:
Done.